src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/BlockSizeRule.cs

```diff
@@ -35,7 +35,7 @@ public override Task RunAsync(RuleContext context)
 
             // Size limits.
             if ((block.Transactions.Count == 0) || (block.Transactions.Count > options.MaxBlockBaseSize) || 
-                (GetSize(this.Parent.Network, block, context.Consensus.NetworkOptions & ~NetworkOptions.All) > options.MaxBlockBaseSize))
+                (GetSize(this.Parent.Network, block, TransactionOptions.All & ~TransactionOptions.All) > options.MaxBlockBaseSize))
```

Comment:
I think `TransactionOptions.All & ~TransactionOptions.All` is equivalent to `TransactionOptions.None`.

Reply:
ah yeah good spot...